Exit if missing lang code - it is required. (#4149)
the-events-calendar/the-events-calendar: readme.txt

= [TBD] TBD =

> * Fix: Fix for scenarios when a lang code is not set in a partial WPML activation, an error would occur and stop processing event saves. [ECP-1442]

Reviewer: ```suggestion
* Fix - Prevent a partial WPML activation, an error would occur and stop processing event saves, when lang code is missing. [ECP-1442]
```
